### ISSUE TITLE
[feature request] shutdown procedure

### ISSUE THREAD

**Issue by papinist** [yojoebosolo/Desktop-Weather#3]
Hi, I just completed my desktop weather ant it is really cool, thank you!
I was wondering if you could add a shutdown procedure: I probably need to move it around the house sometimes, and it's not good to just cut the power while the rpi os is running.
I was thinking of something like "keep the button pressed for 5 seconds", the hands rotate 180° like at start, and the sistem halts. Then after 5\10 seconds, you can remove power.

Do you think it is possible?

**Comment by yojoebosolo** [comment]
Yes absolutely, good idea. I also want to add a couple of other features too so I'll do those at the same time. Thanks - watch this space for the update :)

Just an FYI - I've switched mine off hundreds of times with no issues. Just don't use an expensive SD card and there's not much to worry about. But definitely will implement your suggestion.
